README.md

```diff
@@ -1045,14 +1045,16 @@ Note: in order to see the output of the commands, set the [logging level](#loggi
 
 **Configuration**
 
-| Option              | Description                                                          |
-| ------------------- | -------------------------------------------------------------------- |
-| `mavenCliPath`      | Path to the Maven CLI. It must be executable by the calling process. |
-| `mavenSettingsPath` | Path to the Maven `settings.xml` file.                               |
-| `mavenRepoId`       | ID of the Maven server in the `settings.xml`.                        |
-| `mavenRepoUrl`      | URL of the Maven repository.                                         |
-| `android`           | Android configuration, see below.                                    |
-
+| Option                | Description                                                          |
+| --------------------- | -------------------------------------------------------------------- |
+| `mavenCliPath`        | Path to the Maven CLI. It must be executable by the calling process. |
+| `mavenSettingsPath`   | Path to the Maven `settings.xml` file.                               |
+| `mavenRepoId`         | ID of the Maven server in the `settings.xml`.                        |
+| `mavenRepoUrl`        | URL of the Maven repository.                                         |
+| `android`             | Android configuration, see below.                                    |
+| `kotlinMultiplatform` | Kotlin Multiplatform configuration, see below.                       |
```

Comment:
```suggestion
| `kmp` | Kotlin Multiplatform configuration, see below.                       |
```

Maybe just shorter since there's a description?

Reply:
yup sounds good